Library/Homebrew/cmd/update.sh

```diff
@@ -95,11 +95,13 @@ read_current_revision() {
 
 pop_stash() {
   [[ -z "$STASHED" ]] && return
-  git stash pop "${QUIET_ARGS[@]}"
   if [[ -n "$HOMEBREW_VERBOSE" ]]
   then
+    git stash pop
     echo "Restoring your stashed changes to $DIR:"
```

Comment:
Not that this has been different before this PR, but am I the only one to find it weird that this message is printed after `git stash pop` has already been invoked? And doesn't the following `git status --short --untracked-files` somewhat duplicate the output of `git stash pop`?